### PR TITLE
Add a unit test that makes sure we can add and remove tlc from a channel

### PR DESCRIPTION
**Pull request by contrun**

### CODE REVIEW

File: src/ckb/channel.rs [hunk 1]
```diff
@@ -4554,6 +4564,548 @@ mod tests {
             .expect("accept channel success");
     }
 
+    async fn do_test_channel_commitment_tx_after_add_tlc(algorithm: HashAlgorithm) {
+        let _ = env_logger::try_init();
```
Comment (quake) [inline src/ckb/channel.rs]: please rebase with main branch since env_logger has been removed

File: tests/deploy/contracts/README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+The binaries within this directory are used to verify if the transaction built from our code base can pass ckb-vm verification.
+
+The source code of these binaries are from the following repo [cfn-scripts](https://github.com/nervosnetwork/cfn-scripts) with commit 701d8c8a08790dd61c64f695aaa5fbed22e4b8ad.
+
+We copied the following binaries from https://github.com/nervosnetwork/cfn-scripts/tree/main/deps
+
+- auth
+- simple_udt
+- xudt_rce
```
Comment (quake) [inline tests/deploy/contracts/README.md:12]: there are no xudt_rce binary in cfn-scripts' deps folder.

File: tests/deploy/contracts/auth
Comment (quake) [inline tests/deploy/contracts/auth:1]: we didn't change the auth binary since init commit, why it's changed in this PR?

Comment (contrun) [reply tests/deploy/contracts/auth:1]: I added the wrong binary previously (with some code to print debugging logs). This commit will fix that.

**Review by doitian — APPROVED** [review]

**Review by quake — APPROVED** [review]

**Pull request merged**